test/libyul/objectCompiler/function_series.yul

```diff
@@ -3,6 +3,10 @@ object "Contract" {
     function f() {}
     function g() {}
     sstore(0, 1)
+
+    // NOTE: msize forces the compiler to completely disable the Yul optimizer.
+    // Otherwise the functions would get optimized out.
+    pop(msize())
```

Comment:
This hack is the only way I found to keep this test meaningful. 

The test was added in #6666 and seems to depend on the `sstore` ending up behind the functions so that we have to jump to it. I could not reproduce this with minimal optimizations - the `sstore` always ends up at the beginning.